Godeps

```diff
@@ -3,7 +3,6 @@ github.com/BurntSushi/toml 99064174e013895bbd9b025c31100bd1d9b590ca
 github.com/bmizerany/pat c068ca2f0aacee5ac3681d68e4d0a003b7d1fd2c
 github.com/boltdb/bolt 4b1ebc1869ad66568b313d0dc410e2be72670dda
 github.com/cespare/xxhash 4a94f899c20bc44d4f5f807cb14529e72aca99d6
-github.com/clarkduvall/hyperloglog 2d38f733946d0a1f2e810513c71b834cbeba1480
```

Comment:
It seems like an earlier version of TSI might have used this. There's a package in the `pkg` folder named `hll` that references `hyperloglog` and says it's a faster version. I can't find any direct references to `hyperloglog` though.